internal/js_ast/js_ast.go

```diff
@@ -1043,6 +1043,7 @@ type SNamespace struct {
 type SFunction struct {
 	Fn       Fn
 	IsExport bool
+	HotModule *HotModuleTargets
```

Comment:
In an ideal world, I think a new struct would be the way to go. This approach creates a lot of unused struct fields, but it lets me skip trying to cover all `SFunction` cases.